README.md

```diff
@@ -45,6 +45,8 @@ Jest matcher that performs image comparisons using [Blink-diff](https://github.c
 
 Any blink-diff custom configuration can be provided so long as the values for `imageAPath`, `imageA`, `imageBPath`, `imageB`, or `imageOutputPath` are not changed as these are used internally.
 
+Note that the defaults for `thresholdType` and `threshold` are different than in the blink-diff documentation because we have modified them - our modifications can be viewed [here](https://github.com/americanexpress/jest-image-snapshot/blob/master/src/diff-snapshot.js#L55).
```

Comment:
How about something like instead:

By default [values](https://github.com/americanexpress/jest-image-snapshot/blob/master/src/diff-snapshot.js#L55) are set for `thresholdType` and `threshold`.